src/avro_ocf.erl

```diff
@@ -35,6 +35,8 @@
         , write_header/2
         , write_file/4
         , write_file/5
+        , encode_header/1
+        , make_block/2
```

Comment:
Please sort the export list entries alphanumerically.

Reply:
Fixed